app/views/thredded/private_topic_mailer/message_notification.text.erb

```diff
@@ -1,5 +1,5 @@
 <%= @post.user.email %> said:
-<%= @post.content %>
+<%= @post.filtered_content(self) %>
```

Comment:
Here I think the raw markup is more appropriate, as this is the plain text version